l2_spandex/stratus/project.tcl

```diff
@@ -23,7 +23,8 @@ if {$TECH eq "virtexu"} {
     set_attr default_input_delay      0.1
 }
 if {$TECH eq "virtexup"} {
-    set CLOCK_PERIOD 12.8
+#    set CLOCK_PERIOD 12.8
+    set CLOCK_PERIOD 6.4
```

Comment:
If using a strict clock period fixes the scheduling issues and resolves logic synthesis later on, then perfect.
Please just keep in mind that a strict clock period may result in additional cycles, added during scheduling, for code sections that are not in a protocol region.